packages/react-components/src/components/RichTextEditor/Buttons/Table/insertTableAction.tsx

```diff
@@ -40,3 +44,13 @@ export const insertTable = (editor: IEditor, columns: number, rows: number): voi
     }
   );
 };
+
+function getTableCellWidth(columns: number): string {
+  if (columns <= 4) {
```

Comment:
Should these be in a style?

Reply:
We will delete it as soon as we migrate this file so I wanted to leave all pieces in the same place...